src/frontend/komponenter/App.tsx

```diff
@@ -16,19 +16,28 @@ Modal.setAppElement(document.getElementById('modal-a11y-wrapper'));
 
 interface IState {
     innloggetSaksbehandler?: ISaksbehandler;
+    enhet: string;
```

Comment:
You can add enhet to ISaksbehandler.

Reply:
ISaksbehandler is a literal copy of /user/profile response, which does not contain enhet. If we add enhet to it we have to update it after another REST call (/user/profile), which I find complex to implement.

Reply:
Do you want me to help you?
To get started I would look at this file(https://github.com/navikt/familie-felles-frontend/blob/master/packages/familie-backend/src/auth/passport.ts#L45). In this flow you can add fetching enhet and adding it to the session. 

When this works you can simple get the enhet here: https://github.com/navikt/familie-felles-frontend/blob/master/packages/familie-backend/src/auth/bruker.ts#L14.

Reply:
I just wanted to keep enhet feature independent of the basic user profile, but if you think it is ok I can do it🙂

Reply:
Yes, I think it fits perfectly in the saksbehandler-context. Also, we get away with only one api request 👍